packages/storage-documents/src/privacy-scan-types/privacy-page-scan-result.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { OnDemandPageScanReport, OnDemandPageScanRunResult, WebsiteScanRef } from '../on-demand-page-scan-result';
+import { ItemType } from '../item-type';
+import { StorageDocument } from '../storage-document';
+
+export interface PrivacyPageScanResult extends StorageDocument {
```

Comment:
no need for this type, we will use existing structure with `privacyScan?: boolean;` flag